### ISSUE TITLE
Found Issue with Avoid Edges

### ISSUE THREAD

**Issue by iachaudray** [ecjiang016/Boids-Simulation#1]
Boids don't edges of browser window, instead they avoid edges of the screen. Avoid Edges should be more clearly specified as to what edges they avoid, at this point might as well avoid the edges of the universe.

**Comment by sdey02** [comment]
I agree, I was thoroughly intrigued when I turned on avoid edges and they didn't avoid the right edges....nearly made me want to kms. PLS fix asap
